insert-cover-pages-example/src/parametersDescriptions.ts

```diff
@@ -0,0 +1,90 @@
+export class ParametersDescriptions {
+    static readonly InputParamInputFilePath: InputResourceParameterDescription =
+        {
+            id: 'inputFilePath',
+            displayName: 'Input file path',
+            description: 'Path to TNO file containing source data.',
+            type: 'InputResource',
+            required: true,
+        };
```

Comment:
} as const satisfies InputResourceParameterDescription;

Reply:
added  `as const satisfies ScriptDescription` in `getDescription()_`